src/rpc/names.cpp

```diff
@@ -437,11 +437,15 @@ namespace
 UniValue
 name_show (const JSONRPCRequest& request)
 {
+  bool allow_expired_default = gArgs.GetBoolArg("-allowexpired", DEFAULT_ALLOWEXPIRED);
+
   NameOptionsHelp optHelp;
   optHelp
       .withNameEncoding ()
       .withValueEncoding ()
-      .withByHash ();
+      .withByHash()
+      .withArg("allowExpired", RPCArg::Type::BOOL, allow_expired_default ? "true" : "false",
```

Comment:
Please add a space before the opening parenthesis in the two function calls above, to match the style used in this file.